wire-ios-data-model/Source/Model/Conversation/ZMConversation+SecurityLevel.swift

```diff
@@ -310,11 +310,10 @@ extension ZMConversation {
             appendSystemMessage(type: .participantsAdded, sender: user, users: Set(arrayLiteral: user), clients: nil, timestamp: date)
         case .oneOnOne, .connection:
             if user.connection == nil {
-                user.connection = connection ?? ZMConnection.insertNewObject(in: managedObjectContext!)
-            } else if connection == nil {
-                connection = user.connection
+                user.connection = ZMConnection.insertNewObject(in: managedObjectContext!)
```

Comment:
Couldn't this potentially be a team `oneOnOne` conversation now? Does it make sense to to create `Connection` for team user?

Anyway when can this situation actually happen?

Reply:
Actually, I think it could always have been a team one on one, since the `conversationType` accessor returns `oneOnOne` for team conversations even if the underlying value (in the DB) is for a `group`.

No it doesn't make sense to create a connection for a team user.

We call this method one for every message we receive. 

I suppose we could check first if the user is a team member and if so, don't create the connection.

Reply:
Yes, that should do it.

Reply:
done